.golangci.yml

```diff
@@ -60,3 +57,9 @@ linters-settings:
       - G402
       # G601: Implicit memory aliasing in for loop.
       - G601
+issues:
+  exclude:
+    - "SA1019: \"io/ioutil\" has been deprecated since Go 1.16"
```

Comment:
Ignoring this because Alexc wanted to avoid merge conflicts

https://github.com/argoproj/argo-workflows/pull/6964#issuecomment-946884282

Reply:
Can we add a note here?

Reply:
Note added